pkg/cableengine/syncer/syncer.go

```diff
@@ -30,8 +31,17 @@ type GatewaySyncer struct {
 var GatewayUpdateInterval = 5 * time.Second
 var GatewayStaleTimeout = GatewayUpdateInterval * 3
 
+var gatewaySyncIterations = prometheus.NewCounter(prometheus.CounterOpts{
+	Name: "gateway_sync_iterations",
+	Help: "Gateway synchronisation iterations",
```

Comment:
```suggestion
	Help: "Gateway synchronization iterations",
```

Reply:
Since we pronounce “Submariner” in the British fashion, shouldn’t we use British spelling? :-P (Just kidding, I agree we should use US spelling.)

Reply:
> Since we pronounce “Submariner” in the British fashion, shouldn’t we use British spelling? :-P (Just kidding, I agree we should use US spelling.)

ahh - I didn't know that was British spelling...